### PR TITLE
feat(sdk): verify credential and did wellknown JS api.

### PR DESCRIPTION
**Pull request by vkubiv**

### CODE REVIEW

**Review by birtony — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

File: cmd/wallet-sdk-js/jsinterop/agent.go [hunk 1]
```diff
@@ -177,6 +202,23 @@ func getSubmissionRequirements(_ js.Value, args []js.Value) (any, error) {
 	return util.MapTo(req, types.SerializeMatchedSubmissionRequirement)
 }
 
+func verifyCredentialsStatus(_ js.Value, args []js.Value) (any, error) {
+	if agentInstance == nil {
+		return nil, walleterror.NewExecutionError(
```
Comment (DRK3) [inline cmd/wallet-sdk-js/jsinterop/agent.go:207]: Should use the new "InvalidSDKUsage" method to generate these types of error codes

**Review by talwinder50 — APPROVED** [review]

File: cmd/wallet-sdk-js/jsinterop/agent.go (hunk 1 above)
Comment (DRK3) [reply cmd/wallet-sdk-js/jsinterop/agent.go:207]: `walleterror.NewInvalidSDKUsageError`

File: demo/app/lib/wallet_sdk/wallet_sdk_mobile.dart [hunk 2]
```diff
@@ -167,10 +169,10 @@ class WalletSDK extends WalletPlatform {
     return versionDetailResp;
   }
 
-  Future<Map<Object?, Object?>?> wellKnownDidConfig(String issuerID) async {
-    var didLinkedResp = await methodChannel.invokeMethod('wellKnownDidConfig', <String, dynamic>{'issuerID': issuerID});
-    log("well known config, $didLinkedResp");
-    return didLinkedResp;
+  Future<WellKnownDidConfig> wellKnownDidConfig(String issuerID) async {
```
[the remark this answers is not in the document]
Comment (vkubiv) [reply demo/app/lib/wallet_sdk/wallet_sdk_mobile.dart:172]: Hmm, I actually did it, it looks like it was lost during rebase.  I will add it to PR.

File: demo/app/web/script.js [hunk 3]
```diff
@@ -100,6 +102,19 @@ async function jsVerifierDisplayData() {
...
 }
 
+
+async function jsVerifyCredentialsStatus(credential) {
+    return await agent.verifyCredentialsStatus({
+        credential: credential
+    })
+}
+
+async function jsWellKnownDidConfig(issuerID) {
```
Comment (DRK3) [inline demo/app/web/script.js:112]:
```suggestion
async function jsWellKnownDIDConfig(issuerID) {
```

**Comment by DRK3** [comment]
> feat(sdk): verify credential and did wellknown JS api

Is in the PR title and commit title. It should be

> feat(sdk): verify credential and did wellknown JS API

Instead.

**Pull request merged**